examples/just_enhancement.py

```diff
@@ -150,6 +149,7 @@ def run_scans(nScans, power_idx, DNP_data=None):
         return DNP_data
 #}}}
 ini_time = time.time() # needed b/c data object doesn't exist yet
+power_start_times.append(time.time())
 DNP_data = run_scans(nScans,0)
```

Comment:
and here we store the time for the thermal scan.

Reply:
I will add that I'm a bit confused, because my power_start_times would be 1 longer than yours --> I actually don't understand how the length of yours matches the length of your indirect dimension, and/or where the 0 at the beginning comes from.   So, definitely, you would want to test + debug this with 3 powers before trying a full run.